robosuite/controllers/config/default_baxter.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type" : "JOINT_VEL",
+  "input_max": 1,
+  "input_min": -1,
+  "output_max": 1,
+  "output_min": -1,
+  "kv": [8.0, 7.0, 6.0, 4.0, 2.0, 0.5, 0.1],
+  "velocity_limits": [-1,1],
+  "interpolation": null
```

Comment:
add comma to the end of this line (same for the other jsons)

Reply:
json standard doesn't allow for trailing commas at the end of the dict. Tried and it brought up a parsing error